assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -0,0 +1,38 @@
+[id='assembly_openshift-authoring-managed]
+
+include::_artifacts/document-attributes.adoc[]
+
+= Deploying a {PRODUCT} {PRODUCT_VERSION} authoring or managed server environment on {OPENSHIFT}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: openshift-authoring-managed
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a system engineer, you can deploy a {PRODUCT} authoring or managed environment on {OPENSHIFT} to provide a platform for developing or running {ASSETS} and other business assets.
+
+.Prerequisites
+
+* At least four gigabytes of memory are available in the OpenShift cluster/namespace.
+* The OpenShift project for the deployment is created.
+* You have logged in to the project using the OpenShift web console and using the `oc` command. For more information about the `oc` command-line tool, see the OpenShift https://access.redhat.com/documentation/en-us/openshift_container_platform/3.10/html-single/cli_reference/[CLI Reference].
+* Dynamic persistent volume (PV) provisioning is enabled. Alternatively, enough persistent volumes are available. By default, the following sizes are required:
+** The replicated set of {KIE_SERVER} pods requires one 1Gi PV for the database. This requirement does not apply if you use an external database server.
+** {CENTRAL} requires one 1Gi PV.
+* If you intend to use the Authoring High Availability template, the image streams for Red Hat AMQ version 7.0 or later are available in your OpenShift environment in the `openshift` namespace.
+* If you intend to use the Authoring High Availability template, which scales the {CENTRAL} pod, your OpenShift environment supports persistent volumes with ReadWriteMany mode. For information about access mode support in OpenShift Online volume plug-ins, see https://docs.openshift.com/online/architecture/additional_concepts/storage.html#pv-access-modes[Access Modes].
```

Comment:
the target OpenShift Online environment needs to support ReadWriteMany persistent volumes mode.

Reply:
That would be outright wrong. It is exactly "Your OpenShift environment", it is not necessarily online but whichever one uses. It's just that we provide this info link in case one uses online.